src/components/Filters.js

```diff
@@ -99,47 +99,48 @@ class Filters extends React.Component {
     render() {
     return (
         <section className="section blue">
-            <div className="filters-holder flexy">
+            <div className="card">
+                <h2>How do filters make features easier to see?</h2>
+                <p>Select the different filters in this example. What patterns do you see in the filter values?</p>
+            </div>
+            <div className="filters-holder">
                 <img src={amsterdamRegular} alt="black and white amsterdam" width="500px"></img>
```

Comment:
Ooh, I didn't catch this earlier, but I would avoid hard-coding the `width` as an HTML attribute - it's confusing when combined with CSS classes!